programs/server/config.xml

```diff
@@ -589,6 +589,9 @@
                 toStartOfHour(event_time)
         -->
         <partition_by>toYYYYMM(event_date)</partition_by>
+        <!-- How many days data would be kept. It works only when this option is opened and is set greater than 0.
+        -->
+        <ttl>30</ttl>
```

Comment:
Let's name it `<delete_after_days>` or something, just `<ttl>` is too generic. Maybe we'll want to use it to accept generic TTL specification (or you can do it now if you wish, just paste it into table description).

Reply:
ok, I think we can keep it as `<ttl>` and make it accept generic `TTL` specification.

Reply:
> Let's name it `<delete_after_days>` or something, just `<ttl>` is too generic. Maybe we'll want to use it to accept generic TTL specification (or you can do it now if you wish, just paste it into table description).

ok, I think we can keep it as <ttl> and make it accept generic TTL specification.